.github/workflows/ci-build.yml

```diff
@@ -43,3 +43,6 @@ jobs:
     - env:
         DOCKER_BUILDKIT: 1
       run: docker build .
```

Comment:
We should rename the existing Dockerfile to `Dockerfile.aggregator` and use `-f` here, for consistency and to disambiguate.